web/templates/recipient_row.hbs

```diff
@@ -6,14 +6,9 @@
           style="background: {{background_color}}; border-left-color: {{background_color}};"
           href="{{stream_url}}"
           title="{{#tr}}Narrow to stream &quot;{display_recipient}&quot;{{/tr}}">
-            {{~! Icons for invite-only/web-public streams ~}}
-            {{~#if invite_only ~}}
-                <i class="fa fa-lock recipient-row-stream-icon" title="{{t 'This is a private stream' }}" aria-label="{{t 'This is a private stream' }}"></i>
-            {{/if}}
-
-            {{~#if is_web_public ~}}
-                <i class="zulip-icon zulip-icon-globe recipient-row-stream-icon" title="{{t 'This is a web-public stream' }}" aria-label="{{t 'This is a web-public stream' }}"></i>
-            {{/if}}
+            <span id="stream_sidebar_privacy_swatch_{{stream_id}}" class="stream-privacy filter-icon">
+                {{> stream_privacy}}
+            </span>
```

Comment:
I don't understand this change; `stream_sidebar_privacy_swatch_` seems like it'll be a duplicate ID, and the commit message doesn't explain at all what we're doing, which seems to be removing the (probably unnecessary) `title` tooltips in favor of using the existing left-sidebar component?

That's probably a fine change, but @alya for confirmation that we don't want the tooltips on mousing over the lock/globe symbols in a message feed recipient bar.

Reply:
Sorry, I think this got merged with another commit after recent rebase.